webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsResponderADM.scala

```diff
@@ -780,7 +779,7 @@ class ProjectsResponderADM(responderData: ResponderData) extends Responder(respo
             }
 
             newProjectIRI: IRI = createProjectRequest.projectIri match {
-              case Some(customProjectIri) => stringFormatter.toSmartIriWithErr(customProjectIri, throw BadRequestException(s"Invalid project IRI")).toString
+              case Some(customProjectIri) => stringFormatter.validateProjectIri(customProjectIri, throw BadRequestException(s"Invalid project IRI")).toString
```

Comment:
Could you do this in the route, rather than in the responder, to make things consistent?